silq/instrument_interfaces/keysight/Keysight_SD_AWG_interface.py

```diff
@@ -95,7 +95,8 @@ def active_channel_ids(self):
 
     @property
     def active_instrument_channels(self):
-        return self.instrument.channels[self.active_channel_ids]
+        return [ch for ch in self.instrument.channels for k in self.active_channel_ids if ch.id == k]
+        # return self.instrument.channels[self.active_channel_ids]
```

Comment:
don't really understand this line

Reply:
That's because I wrote it really quick and dirty, I've improved it now.